### DISCUSSION TITLE
Different YAxis Scale

### DISCUSSION THREAD

**Discussion by MatthieuVeillon** [plouc/nivo#2029] [General]
Hi, 

Thanks for the great work, I really enjoy playing with your API.

I'm trying to do a line chart with several lines. The issue I have is that each of those lines can be on a totally different scale in term of values and I would like them to be displayed accordingly their own min/max combination. 

My dataset shape looks like something below. So far I have tried a combination of linear and log scale without any luck. 

Didn't find any reference in the doc about multiples lines with different scales, if you have any ideas on how I could achieve this ? 

````
    {
      id: "UniqueVisitors",
      color: "hsl(179, 70%, 50%)",
      data: [
        {
          x: "april",
          y: 5684,
        },
        {
          x: "mai",
          y: 8478,
        },
      ],
    },
    {
      id: "KeywordsInTop3",
      color: "hsl(290, 70%, 50%)",
      data: [
        {
          x: "april",
          y: 10,
        },
        {
          x: "mai",
          y: 20,
        },
      ],
    },
...
````

**Comment by tkonopka** [comment]
Hi @MatthieuVeillon. This may not be an answer to the technical question you asked, but just wanted to mention that, sometimes, when two parts of a dataset have very different properties, it is reasonable to create two separate simple charts instead of a single complex chart. This approach might be suitable for the data in your example, especially because one of the time series refers to people (visitors) and the other refers to things (keywords). As they carry quite different meaning, why not draw them separately. 

(If your actual data carries different interpretations, the arguement to separate the series can still be made on the grounds of the value distributions alone.)

For this data, you can also consider a scatter-plot with keywords and visitors on two axes (each with their own scales). But whether this makes sense or not will depend on the provenance and meaning of the data you are studying. Hope this helps.
